source/VideoTransportSlider.js

```diff
@@ -646,10 +646,12 @@
 		* @private
 		*/
 		setValue: function(val) {
-			if(Math.abs(this.calcVariationPercent(val)) > this.smallVariation) {
+			var v = this.clampValue(this.min, this.max, val);
+			this.currentTime = v;
```

Comment:
I meant `this.value` - maybe you can verify/double-check this, but it seemed that all of the code pathways originating from `setValue` result in `this.value` being set to the clamped value of the initial value `val`, so I was thinking we could instead just set `this.currentTime = this.value` at the end of the `setValue` function. Let me know what you think, thanks!